docs/NetworkPeeringForScalarDLAuditor.md

```diff
@@ -0,0 +1,62 @@
+# Create network peering for Scalar DL Auditor mode
+
+This document explains how to connect several private networks for ScalarDL Auditor mode. To make ScalarDL Auditor mode work properly, ScalarDL Ledger and ScalarDL Auditor need to connect with each other. 
+
+## Private network you must connect
+
+Depending on your application (client) deployment, you must connect several private networks.
+
+### Connect two private networks
+
+If you deploy your application on the same private network as ScalarDL Ledger or ScalarDL Auditor, you must connect two private networks.
```

Comment:
Reviewed this part in case we decide to keep it or reuse somewhere else.

```suggestion
If you deploy your application (client) on the same private network as ScalarDL Ledger or ScalarDL Auditor, you must connect two private networks.
```